config.yml

```diff
@@ -1043,6 +1045,25 @@ nodes:
           def a(b)
                 ^
           end
+  - name: RescueNode
+    child_nodes:
+      - name: rescue_keyword
+        type: token
+      - name: exception_classes
+        type: node[]
+      - name: exception_variable
```

Comment:
I'd like to capture the `=>` token as well. In general we try to gather up all of the tokens into _some_ node on the tree.